src/Core/Grid/Query/CategoryQueryBuilder.php

```diff
@@ -208,4 +212,15 @@ private function getQueryBuilder(array $filters)
 
         return $qb;
     }
+
+    /**
+     * @param array $filters
+     *
+     * @return bool
+     */
+    private function isSearchRequestOnHomeCategory(array $filters)
+    {
+        return (array_key_exists('is_home_category', $filters) && ($filters['is_home_category'] === true))
```

Comment:
Isset is really faster than array_key_exists (and because we don't care about the null value)